### PR TITLE
Nuke code climate/codacy

### PR DESCRIPTION
**Pull request by shosti**
Seems to be pretty useless, we'll just do custom stuff.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+inherit_gem:
+  rf-stylez: ruby/rubocop.yml
```
Comment (theonegri) [inline .rubocop.yml]: why this change?

Comment (shosti) [reply .rubocop.yml]: This is what it was originally :wink: https://github.com/rainforestapp/circlemator/blob/269080cb69351f9f9c5f995a47585f5478b5199d/.rubocop.yml
